src/pages/reference/preview-features/index.mdx

```diff
@@ -13,7 +13,8 @@ Each preview feature will have its own documentation page, which will include in
 
 | Features                                                | Preview Released | Feedback Issue                                              |
 | ------------------------------------------------------- | ---------------- | ----------------------------------------------------------- |
-| [docker providers](/reference/providers/install/docker) | CLI v1.39.0      | [feedback](https://github.com/nitrictech/nitric/issues/605) |
+| [docker-providers](/reference/providers/install/docker) | CLI v1.39.0      | [feedback](https://github.com/nitrictech/nitric/issues/605) |
+| [sql-databases](/sql)                                   | CLI v1.42.0      | [feedback](https://github.com/nitrictech/cli/issues/732)    |
```

Comment:
might be worth adding the AWS provider version as well, and we can update as we add to more providers